app/src/main/java/com/android/streetworkapp/ui/train/TrainSoloScreen.kt

```diff
@@ -0,0 +1,91 @@
+package com.android.streetworkapp.ui.train
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.android.streetworkapp.model.workout.WorkoutViewModel
+import com.android.streetworkapp.ui.theme.ColorPalette.INTERACTION_COLOR_DARK
+import com.android.streetworkapp.ui.theme.ColorPalette.PRINCIPLE_BACKGROUND_COLOR
+
+@Composable
+fun TrainSoloScreen(
+    activity: String,
+    isTimeDependent: Boolean,
+    workoutViewModel: WorkoutViewModel,
+    paddingValues: PaddingValues = PaddingValues(0.dp)
+) {
+  var isStopped by remember { mutableStateOf(false) }
+
+  Column(
+      modifier = Modifier.fillMaxSize().padding(paddingValues).testTag("TrainSoloScreen"),
+      horizontalAlignment = Alignment.CenterHorizontally,
+      verticalArrangement = Arrangement.Center) {
+        Text("Train Solo", modifier = Modifier.testTag("TrainSoloTitle"))
+        Text("Activity: $activity", modifier = Modifier.testTag("ActivityText"))
+        Text("Time Dependent: $isTimeDependent", modifier = Modifier.testTag("TimeDependentText"))
```

Comment:
You can remove this, I dont think the user need to know.
(but if its for temporary debug then resolve this issue)